### PR TITLE
feat: Add output aws_rds_cluster.this.hosted_zone_id

### PR DESCRIPTION
**Pull request by felixb**
## Description
This allows setting route53 aliases.

## Motivation and Context
I want to set an alias record in route53.

## Breaking Changes
None.

## How Has This Been Tested?
Did not test. It's just a new output.

### CODE REVIEW

**Comment by felixb** [comment]
@antonbabenko any chance of getting this merged and released?

File: outputs.tf [hunk 1]
```diff
@@ -57,6 +57,10 @@ output "this_rds_cluster_instance_ids" {
   value       = aws_rds_cluster_instance.this.*.id
 }
 
+output "this_rds_cluster_hosted_zone_id" {
+  value = aws_rds_cluster.this.hosted_zone_id
```
Comment (bryantbiggs) [inline outputs.tf:61]:
```suggestion
  value = aws_rds_cluster.this[*].hosted_zone_id
```

Comment (bryantbiggs) [inline outputs.tf:61]: @felixb can we add a description for the output

File: outputs.tf [hunk 2]
```diff
@@ -57,6 +57,11 @@ output "this_rds_cluster_instance_ids" {
   value       = aws_rds_cluster_instance.this.*.id
 }
 
+output "this_rds_cluster_hosted_zone_id" {
+  description = "Route53 hosted zone id of the created cluster"
+  value       = aws_rds_cluster.this[*].hosted_zone_id
```
Comment (bryantbiggs) [inline outputs.tf:62]: my apologies @felixb , you were correct before. this is the cluster resource not the cluster instance:
```suggestion
  value       = aws_rds_cluster.this.hosted_zone_id
```

**Comment by bryantbiggs** [comment]
@antonbabenko 👍

**Comment by antonbabenko** [comment]
Thanks, @felixb !

[v2.22.0](https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/releases/v2.22.0) has been just released.

**Pull request merged**